<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>

 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x><y>0</y>
    <width>832</width><height>550</height>
   </rect>
  </property>
  <property name="windowTitle"><string>آوا فارسی</string></property>
  <property name="windowFlags"><set>Qt::FramelessWindowHint</set></property>
  <property name="attribute"><set>Qt::WA_TranslucentBackground</set></property>
  <property name="styleSheet">
   <string notr="true">QWidget{
                        background-color: rgb(65, 65, 65);
                        border-radius: 30px;}
                       QPushButton{
                        background-color: rgb(120, 70, 252);
                        color: white;
                        border-radius: 35px;}
                       QPushButton:hover{
                        background-color: rgb(20, 98, 252);
                        color: white;
                        border-radius: 35px;}
                       QComboBox{
                        border: 2px solid gray;
                        color: white;
                        border-radius: 20px;
                        background-color: rgb(85, 85, 85);}
                       QComboBox::drop-down {
                        border-width: 0px;}
                       QProgressBar {
                        color: white;
                        text-align: center;
                        border-radius: 20px;}
                       QProgressBar::chunk {
                        border-top-left-radius: 20px;
                        border-bottom-left-radius: 20px;
                        background-color:  rgb(120, 70, 252);}
                       QLineEdit {
                        color: gray;
                        border-radius: 20px;
                        border: 2px solid rgb(130, 130, 130);}
   </string>
  </property>

  #Menu Page start
  <widget class="QWidget" name="menu_page">
    <property name="geometry">
     <rect>
      <x>0</x><y>0</y>
      <width>832</width><height>530</height>
     </rect>
    </property>

    <widget class="QPushButton" name="inbox">
    <property name="geometry">
     <rect>
      <x>10</x><y>9</y>
      <width>32</width><height>32</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb();
                         background-image: url(../resources/ui/widgets_img/inbox.png);
     </string>
    </property>
    </widget>

    <widget class="QPlainTextEdit" name="inbox_message">
    <property name="geometry">
     <rect>
      <x>7</x><y>42</y>
      <width>221</width><height>261</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>15</pointsize></font></property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(122, 122, 122);
                         border-radius: 20px;
                         color: white;</string>
    </property>
    <property name="enabled"><bool>true</bool></property>
    <property name="visible"><bool>false</bool></property>
    </widget>

    <widget class="QLabel" name="version_info">
    <property name="geometry">
     <rect>
      <x>10</x><y>508</y>
      <width>50</width><height>12</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>13</pointsize></font></property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb();
                         color: gray;</string>
    </property>
    <property name="text"><string>v2.0.0</string></property>
    </widget>

    <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>271</x><y>30</y>
      <width>199</width><height>51</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>28</pointsize></font></property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text"><string>آوا فارسی</string></property>
    </widget>

    <widget class="QFrame" name="title_line">
    <property name="geometry">
     <rect>
      <x>255</x><y>90</y>
      <width>322</width><height>1</height>
     </rect>
    </property>
    <property name="FrameShape"><set>QFrame::HLine</set></property>
    <property name="FrameShadow"><set>QFrame::Sunken</set></property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 90, 90);</string>
    </property>
    </widget>

    <widget class="QPushButton" name="dl_singer_name">
    <property name="geometry">
     <rect>
      <x>235</x><y>145</y>
      <width>361</width><height>90</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>16</pointsize></font></property>
    <property name="text"><string>دانلود با نام خواننده</string></property>
    </widget>

    <widget class="QPushButton" name="dl_music_name">
    <property name="geometry">
     <rect>
      <x>235</x><y>250</y>
      <width>361</width><height>90</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>16</pointsize></font></property>
    <property name="text"><string>دانلود با نام موزیک</string></property>
    </widget>

    <widget class="QPushButton" name="dl_part_music">
    <property name="geometry">
     <rect>
      <x>235</x><y>355</y>
      <width>361</width><height>90</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>16</pointsize></font></property>
    <property name="text"><string>دانلود با متن بخشی از موزیک</string></property>
    </widget>
   </widget>
   #END Menu Page

   #START Search With Singer name
   <widget class="QWidget" name="search_singer_name">
    <property name="geometry">
     <rect>
      <x>0</x><y>0</y>
      <width>832</width><height>530</height>
     </rect>
    </property>
    <property name="visible"><bool>false</bool></property>

    <widget class="QPushButton" name="search_singer_name_btn">
    <property name="geometry">
     <rect>
      <x>262</x><y>290</y>
      <width>310</width><height>71</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>16</pointsize></font></property>
    <property name="text"><string>جستجو</string></property>
    </widget>

    <widget class="QLineEdit" name="get_singer_name">
    <property name="geometry">
     <rect>
      <x>262</x><y>230</y>
      <width>310</width><height>50</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>16</pointsize></font></property>
    <property name="placeholderText"><string>   نام خواننده را اینجا بنویسید</string></property>
    </widget>

    <widget class="QPushButton" name="live_face">
    <property name="geometry">
     <rect>
      <x>580</x><y>229</y>
      <width>50</width><height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
    <string notr="true">background-image: url(../resources/ui/widgets_img/unknown_face.png);
                        border-radius: 15px;
    </string>
    </property>
    </widget>

    <widget class="QLabel" name="notice_0">
    <property name="geometry">
     <rect>
      <x>262</x><y>190</y>
      <width>310</width><height>30</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>14</pointsize></font></property>
    <property name="alignment"><set>Qt::AlignHCenter</set></property>
    <property name="styleSheet">
    <string notr="true">color: rgb(255, 37, 21);</string>
    </property>
    </widget>

    <widget class="QRadioButton" name="first_site">
    <property name="geometry">
     <rect>
      <x>110</x><y>200</y>
      <width>131</width><height>41</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>15</pointsize></font></property>
    <property name="text"><string>MrTehran</string></property>
    <property name="checked"><bool>true</bool></property>
    <property name="styleSheet">
    <string notr="true">QRadioButton {
                         color :rgb(165, 165, 165);}
                        QRadioButton::indicator {
                         width: 25px;
                         height: 25px;}</string>
    </property>
    </widget>

    <widget class="QRadioButton" name="second_site">
    <property name="geometry">
     <rect>
      <x>110</x><y>234</y>
      <width>131</width><height>41</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>15</pointsize></font></property>
    <property name="text"><string>MusicFa</string></property>
    <property name="styleSheet">
    <string notr="true">QRadioButton {
                         color :rgb(165, 165, 165);}
                        QRadioButton::indicator {
                         width: 25px;
                         height: 25px;}</string>
    </property>
    </widget>

    <widget class="QRadioButton" name="first_site">
    <property name="geometry">
     <rect>
      <x>110</x><y>272</y>
      <width>131</width><height>41</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>15</pointsize></font></property>
    <property name="text"><string>UpMusics</string></property>
    <property name="styleSheet">
    <string notr="true">QRadioButton {
                         color :rgb(165, 165, 165);}
                        QRadioButton::indicator {
                         width: 25px;
                         height: 25px;}</string>
    </property>
    </widget>
   </widget>
   # END Search Singer Name

   #START Download with Singer Name(page1)
   <widget class="QWidget" name="page_1">
    <property name="geometry">
     <rect>
      <x>0</x><y>0</y>
      <width>832</width><height>530</height>
     </rect>
    </property>
    <property name="visible"><bool>false</bool></property>

    <widget class="QLabel" name="notice_1">
    <property name="geometry">
     <rect>
      <x>130</x><y>6</y>
      <width>571</width><height>51</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>20</pointsize></font></property>
    <property name="alignment"><set>Qt::AlignHCenter</set></property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 37, 21);</string>
    </property>
    </widget>

    <widget class="QLabel" name="single_dl_label">
    <property name="geometry">
     <rect>
      <x>70</x><y>50</y>
      <width>321</width><height>51</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>20</pointsize></font></property>
    <property name="alignment"><set>Qt::AlignHCenter</set></property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text"><string>دانلود تکی</string></property>
    </widget>

    <widget class="QComboBox" name="select_music_name">
    <property name="geometry">
     <rect>
      <x>70</x><y>100</y>
      <width>321</width><height>50</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>14</pointsize></font></property>
    <property name="layoutDirection"><set>Qt::RightToLeft</set></property>
    </widget>

    <widget class="QPushButton" name="single_dl_btn">
    <property name="geometry">
     <rect>
      <x>70</x><y>160</y>
      <width>321</width><height>60</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>30</pointsize></font></property>
    <property name="text"><string>دانلود</string></property>
    <property name="styleSheet"><string>border-radius: 25px;</string></property>
    </widget>

    <widget class="QLabel" name="multiple_dl_label">
    <property name="geometry">
     <rect>
      <x>70</x><y>224</y>
      <width>321</width><height>51</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>20</pointsize></font></property>
    <property name="alignment"><set>Qt::AlignHCenter</set></property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text"><string>دانلود چند تایی</string></property>
    </widget>

    <widget class="QLineEdit" name="multiple_dl_input">
    <property name="geometry">
     <rect>
      <x>70</x><y>275</y>
      <width>321</width><height>50</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>14</pointsize></font></property>
    <property name="placeholderText"><string>   تعداد موزیک هایی که میخواهید دانلود شود</string></property>
    </widget>

    <widget class="QPushButton" name="multiple_dl_btn">
    <property name="geometry">
     <rect>
      <x>70</x><y>335</y>
      <width>321</width><height>60</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>30</pointsize></font></property>
    <property name="text"><string>دانلود</string></property>
    <property name="styleSheet"><string>border-radius: 25px;</string></property>
    </widget>

    <widget class="QPushButton" name="dl_all_musics">
    <property name="geometry">
     <rect>
      <x>70</x><y>420</y>
      <width>321</width><height>71</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>15</pointsize></font></property>
    <property name="text"><string>دانلود تمام موزیک ها به صورت یکجا</string></property>
    </widget>

    <widget class="QWidget" name="singer_img">
    <property name="geometry">
     <rect>
      <x>470</x><y>55</y>
      <width>340</width><height>340</height>
     </rect>
    </property>
    </widget>

    <widget class="QWidget" name="singer_info">
    <property name="geometry">
     <rect>
      <x>470</x><y>420</y>
      <width>340</width><height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
    <string notr="true">background-color: rgb(113, 113, 113);
                        border-radius: 30px;
    </string>
    </property>

    <widget class="QLabel" name="singer_name">
    <property name="geometry">
     <rect>
      <x>10</x><y>7</y>
      <width>321</width><height>41</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>20</pointsize></font></property>
    <property name="alignment"><set>Qt::AlignHCenter</set></property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    </widget>

    <widget class="QLabel" name="number_of_musics">
    <property name="geometry">
     <rect>
      <x>20</x><y>40</y>
      <width>301</width><height>24</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>13</pointsize></font></property>
    <property name="alignment"><set>Qt::AlignHCenter</set></property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    </widget>
    </widget>

    <widget class="QProgressBar" name="downloadBar_1">
    <property name="geometry">
     <rect>
      <x>0</x><y>0</y>
      <width>0</width><height>0</height>
     </rect>
    </property>
    </widget>
   </widget>
   #END Download with Singer Name(page1)

   #START search by muisc name and part of music(page2)
   <widget class="QWidget" name="search_music_name">
    <property name="geometry">
     <rect>
      <x>0</x><y>0</y>
      <width>832</width><height>530</height>
     </rect>
    </property>
    <property name="visible"><bool>false</bool></property>

    <widget class="QLabel" name="notice_2">
    <property name="geometry">
     <rect>
      <x>262</x><y>190</y>
      <width>310</width><height>30</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>14</pointsize></font></property>
    <property name="alignment"><set>Qt::AlignHCenter</set></property>
    <property name="styleSheet">
    <string notr="true">color: rgb(255, 37, 21);</string>
    </property>
    </widget>

    <widget class="QLineEdit" name="music_name">
    <property name="geometry">
     <rect>
      <x>262</x><y>230</y>
      <width>310</width><height>50</height>
     </rect>
    </property>
    <property name="font"><font><pointsize>16</pointsize></font></property>
    <property name="placeholderText"><string>   نام موزیک را اینجا بنویسید</string></property>
    </widget>

    <widget class="QPushButton" name="search_music_name_btn">
    <property name="geometry">
     <rect>
      <x>262</x><y>290</y>
      <width>310</width><height>71</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>16</pointsize></font></property>
    <property name="text"><string>جستجو</string></property>
    </widget>

    <widget class="QComboBox" name="select_music">
    <property name="geometry">
     <rect>
      <x>262</x><y>230</y>
      <width>310</width><height>50</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>14</pointsize></font></property>
    <property name="layoutDirection"><set>Qt::RightToLeft</set></property>
    <property name="visible"><bool>false</bool></property>
    </widget>

    <widget class="QPushButton" name="dl_music_name_btn">
    <property name="geometry">
     <rect>
      <x>262</x><y>290</y>
      <width>310</width><height>71</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="font"><font><pointsize>18</pointsize></font></property>
    <property name="text"><string>دانلود</string></property>
    <property name="visible"><bool>false</bool></property>
    </widget>


    <widget class="QProgressBar" name="downloadBar_2">
    <property name="geometry">
     <rect>
      <x>262</x><y>290</y>
      <width>310</width><height>71</height>
     </rect>
    </property>
    <property name="visible"><bool>false</bool></property>
    </widget>
   </widget>
   #END search by muisc name and part of music(page2)

   #START Top Buttons
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x><y>10</y>
      <width>101</width><height>33</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="styleSheet">
    <string notr="true">QPushButton{
                         border-radius: 10px;
                         background-color: rgb(20, 98, 252);
                         color: white;
                         background-image: url(../resources/ui/widgets_img/purple_back.PNG);}
                        QPushButton:hover{
                         border-radius: 30px;
                         background-color: rgb(20, 98, 252);
                         color: white;
                         background-image: url(../resources/ui/widgets_img/blue_back.PNG);}
    </string>
    </property>
    <property name="visible"><bool>false</bool></property>
  </widget>

  <widget class="QPushButton" name="close_window">
    <property name="geometry">
     <rect>
      <x>805</x><y>13</y>
      <width>16</width><height>16</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="styleSheet">
    <string notr="true">QPushButton{
                         border-radius: 8px;
                         background-color: rgb(255, 0, 0);
                         color: white;}
                        QPushButton:hover{
                         border-radius: 8px;
                         background-color: rgb(200, 0, 0);
                         color: white;}
    </string>
    </property>
  </widget>

  <widget class="QPushButton" name="minimize">
    <property name="geometry">
     <rect>
      <x>785</x><y>13</y>
      <width>16</width><height>16</height>
     </rect>
    </property>
    <property name="cursor"><cursorShape>PointingHandCursor</cursorShape></property>
    <property name="styleSheet">
    <string notr="true">QPushButton{
                         border-radius: 8px;
                         background-color: rgb(255, 255, 0);
                         color: white;}
                        QPushButton:hover{
                         border-radius: 8px;
                         background-color: rgb(200, 200, 0);
                         color: white;}
    </string>
    </property>
  </widget>
  #end Top Buttons
 </widget>
</ui>